src/common/strlib.c

```diff
@@ -1068,16 +1068,35 @@ bool sv_readdb(const char* directory, const char* filename, char delim, int minc
 /// Allocates a StringBuf
 StringBuf* StringBuf_Malloc()
 {
+	return StringBuf_MallocInitial(1024);
```

Comment:
is there a reason why 1024 as initial ? it's quite a long string to start with no ?